pcsx2/DebugTools/Debug.h

```diff
@@ -73,7 +73,7 @@ class SysTraceLog : public TextFileTraceLog
 
 	// Pass me a NULL and you *will* suffer!  Muahahaha.
 	SysTraceLog( const SysTraceLogDescriptor* desc )
-		: TextFileTraceLog( &desc->base ) {}
+		: TextFileTraceLog( &desc->base ), PrePrefix(NULL) {}
```

Comment:
I don't think PrePrefix is used anywhere. Might be a better idea to remove it with the SetPrefix function.